doc/customize.md

```diff
@@ -0,0 +1,449 @@
+about `No such file or directory`. These messages can be safely
+ignored. See also <https://github.com/damianavila/RISE/issues/353> about
+this.
+
+How to customize
+----------------
+
+RISE can be customized in a lot of ways. As of RISE version 5.3, you can:
+
+1. use `nbextensions_configurator`; this tool offers an interactive
+   way to enable, disable and tweak all notebook extensions - see
+   screenshot below;
+
+1. you can also embed settings in a specific notebook's metadata;
+
+1. and you can also provide your own css file(s), that can supersede
+   styling of the various DOM pieces.
+
+### The configurator
+
+You may need to install a separate module:
+
+    pip3 install jupyter-nbextensions-configurator
+
+You should then see a fourth tab in jupyter's directory views, as
+depicted below. Settings are stored in JSON format, typically in
+
+    ~/.jupyter/nbconfig/notebook.json
+
+![](../examples/configurator.png)
+
+
+### Notebook metadata
+
+These settings can also be stored in your notebook metadata, which
+holds a JSON object, that can be edited through Jupyter's standard
+menu (Edit → Edit Notebook Metadata); typically it would look like
+this:
+
+    {
+        ...
+        "rise": {
+            "theme": "serif",
+            "transition": "zoom",
+            ...
+        },
+        ...
+    }
+You can edit notebook metadata as follows
+
+![](../examples/metadata.png)
+
+### Note on legacy naming
+
+In all this document we store settings in a JSON key named `rise`.
+You may also see some notebooks using the `livereveal` key instead,
+which is an older name for the same project. Both names are actually
+taken into account, however you should know that `rise` will take
+precedence on `livereveal` if the same setting in defined under both
+names.
+
+### Order of precedence
+
+If we forget about custom CSS for now, that is to say as far as the
+first 2 categories are concerned: at this point you need to be aware
+that:
+
+* settings changed through the configurator - or using python as we
+  will see below - are stored on your own file system, typically in
+  your home directory, and so are only be applicable to you;
+
+* *a contrario* settings embedded in a specific notebook's metadata
+   will be applicable to all users, even if they end up in a mybinder
+   instance.
+
+Apart from that, the scope of what is configurable through both
+channels (configurator and metadata) is identical, so it is possible
+to use the configurator as some sort of an online reference manual,
+as it describes each and every setting.
+
+Finally, the following priorities apply:
+
+* a setting will always be used if you define it in the `rise` section
+  of the nodebook metadata;
+
+* if not, it will be used if it is present in the legacy `livereveal`
+  (see below) section of the notebook metadata;
+
+* then if not, the settings from your own profile (either defined
+  through the configurator, or through python) are applied.
```

Comment:
In this section, you are making reference to the config with python resulting in a `rise.json` file.
I am surprised to see this reference given that you said you need to known more about this `rise.json` thing.